src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1803,7 +1803,7 @@ java_bytecode_parsert::parse_method_handle(const method_handle_infot &entry)
 void java_bytecode_parsert::read_bootstrapmethods_entry(classt &parsed_class)
 {
   u2 num_bootstrap_methods = read_u2();
-  for(size_t i = 0; i < num_bootstrap_methods; i++)
+  for(size_t j = 0; j < num_bootstrap_methods; j++)
```

Comment:
Why has this changed from `i` to `j`

Reply:
Because it was colliding with another loop, perhaps a good argument for a better name like `boostrap_method_index`?